tcs-service/src/test/resources/db/migration/schema/V4.25__create_conditions_of_joining_table.sql

```diff
@@ -0,0 +1 @@
+/* The uuid field is not available on ProgrammeMembership, so we cannot test this migration. */
```

Comment:
I think I might need you to explain this one to me.  `ProgrammeMembership.uuid` exists in the schema but I don't know why it is not available.

Reply:
When the integration tests run this migration fails as it can't find the `ProgrammeMembership.uuid` in the test database (H2), I had assumed it was due to `v4.21` being commented out too
https://github.com/Health-Education-England/TIS-TCS/blob/ee142daef286ddc2945c5afb7482d8ed1b7090fa/tcs-service/src/test/resources/db/migration/schema/V4.21__refactor_programmeMembership.sql